packit_service/worker/helpers/testing_farm.py

```diff
@@ -495,29 +618,61 @@ def run_testing_farm(
                 success=True,
                 details={"msg": "Project not allowed to use internal TF."},
             )
+
+        additional_build = None
+        if self.copr_builds_from_other_pr and not (
+            additional_build := self.copr_builds_from_other_pr.get(chroot)
+        ):
+            self.report_status_to_test_for_test_target(
+                state=BaseCommitStatus.failure,
+                description="No latest successful Copr build from the other PR found.",
+                target=target,
```

Comment:
what would be the target name for these special builds?

Reply:
for now, I haven't changed the check's names so they would not reflect that the tests are run with the builds from other PR...but I was also thinking about including the pr argument somehow, but I am not sure whether to go that direction from the beginning...WDYT?

Reply:
I am sure there will be a discussion (and issues created) about the naming once this feature is live.

I'd go for keeping the name equivalent, as you implemented it, and treat it as a future RFE if people want it configurable or have a different default.

Would that work for you?

Reply:
agreed :+1: